### PR TITLE
Move away monitoring setup

### PR DESCRIPTION
**Pull request by k32**

### CODE REVIEW

File: cdk_emqx_cluster/cdk_emqx_cluster_stack.py [hunk 1]
```diff
@@ -419,17 +425,18 @@ def setup_monitoring(self, targets):
...
-                                            'ghcr.io/emqx/perf-test-postgres:master'),
+                                            'ghcr.io/k32/sysmon-postgres:0.1.1'),
                                         port_mappings=[
                                             ecs.PortMapping(container_port=5432)],
                                         stop_timeout=Duration.seconds(10), # It looks like postgres doesn't want to die sometimes
                                         start_timeout=Duration.seconds(300),
                                         environment={
-                                            'POSTGRES_PASSWORD': '123',
+                                            'POSTGRES_PASSWORD': self.postgresPass,
+                                            'SYSMON_PASS': self.postgresPass,
```
Comment (savonarola) [inline cdk_emqx_cluster/cdk_emqx_cluster_stack.py:438]: Password reuse is a bit confusing 🤔

File: cdk_emqx_cluster/cdk_emqx_cluster_stack.py [hunk 2]
```diff
@@ -451,7 +458,10 @@ def setup_monitoring(self, targets):
         c_grafana = task.add_container('grafana',
                                        essential=True,
                                        image=ecs.ContainerImage.from_registry(
-                                           'ghcr.io/emqx/perf-test-grafana:master'),
+                                           'ghcr.io/k32/sysmon-grafana:0.1.1'),
+                                       environment={
+                                           'POSTGRES_PASS': self.postgresPass
```
Comment (savonarola) [inline cdk_emqx_cluster/cdk_emqx_cluster_stack.py:463]: Sometimes it's `POSTGRES_PASS` and sometimes `POSTGRES_PASSWORD`, is it different parameter naming in different containers?

Comment (k32) [reply cdk_emqx_cluster/cdk_emqx_cluster_stack.py:463]: Yes, correct.

File: cdk_emqx_cluster/cdk_emqx_cluster_stack.py (hunk 1 above)
Comment (k32) [reply cdk_emqx_cluster/cdk_emqx_cluster_stack.py:438]: Having different passwords for the different users allows to give grafana only a read-only access, and restrict access for system_monitor. This makes sense in more sophisticated environments where monitoring infra and the nodes are controlled by different parties. This is not the case in our test environment.

**Review by thalesmg — APPROVED** [review]

**Review by savonarola — APPROVED** [review]

**Pull request merged**